.github/workflows/typescript.yml

```
@@ -1,25 +1,64 @@
-name: TypeScript
+name: TypeScript CI/CD
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)